common/brave_channel_info.cc

```diff
@@ -23,13 +23,4 @@ std::string GetChannelName() {
 #endif  // !OFFICIAL_BUILD
 }
 
-bool IsNightlyOrDeveloperBuild() {
```

Comment:
should this be part of this PR?

Reply:
This is related to adding variations as field trials replace the need for custom ad notifications to be pinned to nightly.